scenarios/kubernetes_verify.py

```diff
@@ -62,7 +68,7 @@ def main(branch, script, force):
     remote = 'bootstrap-upstream'
     uri = 'https://github.com/kubernetes/kubernetes.git'
     try:
-        check('git', 'remote', 'remove', remote)
+        check_output('git', 'remote', 'remove', remote)
```

Comment:
OK, I've updated this to look at the output of `git remote` to see if the bootstrap-upstream remote exists.

Using `git remote show bootstrap-upstream` is non-ideal, since it actually queries the remote. With `show -n`, the remote isn't queried, but then there's no indication from the output whether the remote is known.